docker/hub/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+cd $(git rev-parse --show-toplevel)
+
+# go run ./cmd/generate-keys --private-key=docker/hub/config/matrix_key.pem -tls-cert=docker/hub/config/server.crt -tls-key=docker/hub/config/server.key
+
+docker build -f docker/hub/Dockerfile -t matrixdotorg/dendrite:latest .
+
+COMPONENTS=$(ls docker/hub/Dockerfile.* | cut -d "." -f2)
+for NAME in $COMPONENTS; do
+	docker build -f docker/hub/Dockerfile.$NAME -t matrixdotorg/dendrite:$NAME .
```

Comment:
This doesn't allow any versioning. Also, it's unclear that `Dockerfile.roomserver` maps to `matrixdotorg/dendrite:roomserver`, it's surprising behaviour.